### PR TITLE
fix doc incorrect section name

### LINKED ISSUE #58036: docs: The title of the section on the tutorial page does not match the content
### Describe the problem that you experienced

When navigating to this page: https://angular.dev/tutorials/first-app/11-details-page, I noticed that the title of the first section reads "Create a new service for your app." However, the content of this section doesn't discuss creating a service. Instead, it talks about fetching the dynamic "id" value from the route.

It seems like the title may need updating to reflect the actual content.

### Enter the URL of the topic with the problem

https://angular.dev/tutorials/first-app/11-details-page

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

_No response_

### Add a screenshot if that helps illustrate the problem

image

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by StankaKopalova**
fixes #58036 

- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
- [x] Documentation content changes

## What is the current behavior?

Issue Number: N/A
#58036 

## What is the new behavior?
changed wrong title of section in documentation

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by MarkTechson** [comment]
@StankaKopalova this is a great find, can you please resolve the merge conflicts in your branch?

File: aio/content/tutorial/first-app/first-app-lesson-11.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Each housing location has specific details that should be displayed when a user
 
 Route parameters enable you to include dynamic information as a part of your route URL. To identify which housing location a user has clicked on you will use the `id` property of the `HousingLocation` type.
 
-## Step 1 - Create a new service for your app
+## Step 1 - Fetching the dynamic value from the route
```
Comment (MarkTechson) [inline aio/content/tutorial/first-app/first-app-lesson-11.md:19]: Specifically, this step is about adding route parameters, so this is probably a better title?

```suggestion
## Step 1 - Add route parameters
```

**Review by MarkTechson — CHANGES_REQUESTED** [review]
If you are able to fix the merge conflict and make the change I suggested, we should be able to move this one forward.

**Comment by JeanMeche** [comment]
This a change on a file that doesn't exist anymore.
For changes on angular.dev, please make sure to edit files in the `adev` directory.

**Pull request closed without merging**